x/profile/types/codec.go

```diff
@@ -11,12 +11,15 @@ import (
 
 func RegisterCodec(cdc *codec.LegacyAmino) {
 	// this line is used by starport scaffolding # 2
+	cdc.RegisterConcrete(&MsgUpdateCoordinatorDescription{}, "profile/UpdateCoordinatorDescription", nil)
 	cdc.RegisterConcrete(&MsgCreateCoordinator{}, "profile/CreateCoordinator", nil)
-
 }
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	// this line is used by starport scaffolding # 3
+	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
All registration are separated to make the file modification easier from Starport but we can have:
```
registry.RegisterImplementations((*sdk.Msg)(nil),
    &MsgCreateCoordinator{},
    &MsgUpdateCoordinatorDescription{},
)
```